test/integration_helper.rb

```diff
@@ -63,7 +63,7 @@ def installed?
 
     def command
       @command ||= begin
-        cmd = ["APP_ENV=#{environment}", "exec"]
+        cmd = ["APP_ENV=#{environment}", "bundle", "exec"]
```

Comment:
Okay, this isn't needed when the tests run, because `bundle exec` is used (`bundle exec rake`). It is just when you copy the command to your shell and try to run it by hand it is a problem.